src/push/index.ts

```diff
@@ -78,7 +78,7 @@ export async function push(monitors: Monitor[], options: PushOptions) {
 
   const { monitors: remote } = await bulkGetMonitors(options);
 
-  progress(`bundling ${monitors.length} monitors`);
+  progress(`preparing ${monitors.length} monitors`);
```

Comment:
reworded this as this was confusing as we also include LW monitors here which doesn't get bundled.